src/views/private/admin-leaderboard/leaderboard.js

```diff
@@ -1,65 +1,113 @@
-import React, { useState, useEffect } from 'react';
-import { connect } from 'react-redux';
-import { fetchTeams, editTeamName } from '../actions/actions';
-import TeamCard from './teamCard';
-import AddTeamButton from './addTeam';
-import { Container, Title, BlueFlex, AddTeam } from '../styled-components/leaderboard-styles';
+import React, { useState, useEffect } from "react";
+import { Collapse, Button, CardBody, Card } from "reactstrap";
+import { connect } from "react-redux";
+import { fetchTeams, editTeamName, fetchAllUsers } from "../actions/actions";
+import TeamCard from "./teamCard";
+import AddTeamButton from "./addTeam";
+import {
+  Container,
+  Title,
+  BlueFlex,
+  AddTeam,
+} from "../styled-components/leaderboard-styles";
 
 const Leaderboard = (props) => {
-    const [render, setRender] = useState(false)
+  const [render, setRender] = useState(false);
 
+  //state for accordion function
+  const [teamIndex, setTeamIndex] = useState(null);
 
-    useEffect(() => {
-        props.fetchTeams()
-    }, [render])
+  useEffect(() => {
+      props.fetchTeams();
+      props.fetchAllUsers();
```

Comment:
nice job adding fetching all users to ensure all users are pulled in from backend on render of this component